src/schemas/player.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Player",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "role": {
+      "enum": [
+        "WR",
+        "TE",
+        "QB",
+        "RB",
+        "K"
+      ]
+    },
+    "scores": {
+      "type": "object",
+      "properties": {
+        "catches": {
+          "type": "number"
+        },
+        "recYds": {
+          "type": "number"
+        },
+        "recTouchdowns": {
+          "type": "number"
+        },
+        "rushTouchdowns": {
+          "type": "number"
+        },
+        "rushYds": {
+          "type": "number"
+        },
+        "passYds": {
+          "type": "number"
+        },
+        "passTouchdowns": {
+          "type": "number"
+        },
+        "fumbles": {
+          "type": "number"
+        },
+        "fumblesLost": {
```

Comment:
I think they don't differentiate between a fumble whether it's been lost or recovered, so no need for this I don't think

Reply:
Removed